# Change Log

## [Unreleased](https://github.com/sanger-pathogens/Bio-VertRes-Config/tree/HEAD)

**Merged pull requests:**

- Update README [\#126](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/126) ([ssjunnebo](https://github.com/ssjunnebo))
- Rt641530 unix group [\#125](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/125) ([vaofford](https://github.com/vaofford))
- Rt641530 unix group [\#124](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/124) ([ssjunnebo](https://github.com/ssjunnebo))
- add --no\_anntotation option to pacbio\_register [\#123](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/123) ([vaofford](https://github.com/vaofford))
- Update usage messages [\#122](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/122) ([ssjunnebo](https://github.com/ssjunnebo))
- annotate canu assemblies [\#121](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/121) ([andrewjpage](https://github.com/andrewjpage))
- increas genome size to 7mb [\#120](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/120) ([andrewjpage](https://github.com/andrewjpage))
- change tophat version to tophat-2.1.1 [\#119](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/119) ([vaofford](https://github.com/vaofford))
- dont have version in the alias for hgap [\#118](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/118) ([andrewjpage](https://github.com/andrewjpage))
- annotate HGAP 4.0 assemblies [\#117](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/117) ([andrewjpage](https://github.com/andrewjpage))
- 585953 mlwarehouse [\#116](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/116) ([ssjunnebo](https://github.com/ssjunnebo))
- rename license file [\#114](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/114) ([ssjunnebo](https://github.com/ssjunnebo))
- update LICENSE and AUTHORS [\#113](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/113) ([ssjunnebo](https://github.com/ssjunnebo))
- change bwa version to 0.7.15 [\#112](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/112) ([ssjunnebo](https://github.com/ssjunnebo))
- Turn off store pipeline [\#111](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/111) ([andrewjpage](https://github.com/andrewjpage))
- Set spades options [\#110](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/110) ([andrewjpage](https://github.com/andrewjpage))
- infgen [\#109](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/109) ([andrewjpage](https://github.com/andrewjpage))
- move from scratch108 to 118 [\#108](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/108) ([andrewjpage](https://github.com/andrewjpage))
- update to bowtie 2.3.0 [\#107](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/107) ([vaofford](https://github.com/vaofford))
- set careful and cov cutoff [\#106](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/106) ([andrewjpage](https://github.com/andrewjpage))
- allow for spades assemblies to be annotated [\#105](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/105) ([andrewjpage](https://github.com/andrewjpage))
- change pacbio defaults [\#104](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/104) ([andrewjpage](https://github.com/andrewjpage))
- default assembler [\#103](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/103) ([andrewjpage](https://github.com/andrewjpage))
- script to just create permissions files [\#102](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/102) ([andrewjpage](https://github.com/andrewjpage))
- create permissions pipeline files by default for each study [\#101](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/101) ([andrewjpage](https://github.com/andrewjpage))
- annotation default ram 5GB [\#100](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/100) ([andrewjpage](https://github.com/andrewjpage))
- different annotation prefix for non velvet assemblers [\#99](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/99) ([andrewjpage](https://github.com/andrewjpage))
- update spades bacteria pipeline and version [\#98](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/98) ([andrewjpage](https://github.com/andrewjpage))
- change to dist zilla starter bundle [\#97](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/97) ([nds](https://github.com/nds))
- dont set umask for store lane [\#96](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/96) ([andrewjpage](https://github.com/andrewjpage))
- update usage [\#95](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/95) ([andrewjpage](https://github.com/andrewjpage))
- Bacteria rnaseq [\#94](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/94) ([andrewjpage](https://github.com/andrewjpage))
- check if sequencescape didnt return study name, dont allow spaces in â€¦ [\#93](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/93) ([andrewjpage](https://github.com/andrewjpage))
- pacbio studies always registered in pacbio database [\#92](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/92) ([andrewjpage](https://github.com/andrewjpage))
- change default rna seq protocol for bacteria [\#91](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/91) ([nds](https://github.com/nds))
- update default [\#90](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/90) ([andrewjpage](https://github.com/andrewjpage))
- fixing assembler name and prefix [\#89](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/89) ([nds](https://github.com/nds))
- change default assembler name to be pacbio [\#88](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/88) ([nds](https://github.com/nds))
- NDS pacbio study config generation [\#87](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/87) ([nds](https://github.com/nds))
- update bwa version to 0.7.12 [\#86](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/86) ([andrewjpage](https://github.com/andrewjpage))
- Add permissions support [\#85](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/85) ([andrewjpage](https://github.com/andrewjpage))
- Nds tophat frstrand option for rnaseq [\#84](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/84) ([nds](https://github.com/nds))
- Nds add new tophat version [\#83](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/83) ([nds](https://github.com/nds))
- Add iva and iva\_qc params [\#82](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/82) ([nds](https://github.com/nds))
- set a limit on mapping jobs [\#81](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/81) ([andrewjpage](https://github.com/andrewjpage))
- Change log location [\#80](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/80) ([nds](https://github.com/nds))
- Use test database connections file when testing [\#79](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/79) ([nds](https://github.com/nds))
- Add TravisCI support [\#78](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/78) ([bewt85](https://github.com/bewt85))
- update default kraken db [\#77](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/77) ([andrewjpage](https://github.com/andrewjpage))
- warehouse host updated [\#76](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/76) ([aslett1](https://github.com/aslett1))
- Optimise bacteria assembly [\#75](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/75) ([andrewjpage](https://github.com/andrewjpage))
- use latest samtools head [\#74](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/74) ([andrewjpage](https://github.com/andrewjpage))
- Cram import [\#73](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/73) ([andrewjpage](https://github.com/andrewjpage))
- cram import and pacbio import confs [\#72](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/72) ([andrewjpage](https://github.com/andrewjpage))
- add adapter and primer removal options [\#71](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/71) ([martinghunt](https://github.com/martinghunt))
- Removed leftover debugging statement [\#70](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/70) ([carlacummins](https://github.com/carlacummins))
- change regex for rna seq for bam [\#69](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/69) ([andrewjpage](https://github.com/andrewjpage))
- Rnaseq should not create intergenic or coverage plots for everything bar... [\#68](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/68) ([andrewjpage](https://github.com/andrewjpage))
- Added AUTHORS file [\#67](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/67) ([aslett1](https://github.com/aslett1))
- Added dynamic calculation of pipeline version [\#66](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/66) ([carlacummins](https://github.com/carlacummins))
- Add kraken\_db to QC conf files [\#65](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/65) ([martinghunt](https://github.com/martinghunt))
- increase assembly limit to 1000 [\#64](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/64) ([andrewjpage](https://github.com/andrewjpage))
- allow for multiple assemblies [\#63](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/63) ([andrewjpage](https://github.com/andrewjpage))
- Fix failing tests [\#62](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/62) ([andrewjpage](https://github.com/andrewjpage))
- Added option to omit assembly and annotation configs for bacteria\_register\_and\_qc [\#61](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/61) ([carlacummins](https://github.com/carlacummins))
- Add iva assembler [\#60](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/60) ([martinghunt](https://github.com/martinghunt))
- Add IVA assembler [\#59](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/59) ([martinghunt](https://github.com/martinghunt))
- Updated help text [\#58](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/58) ([carlacummins](https://github.com/carlacummins))
- comment out config files at the top level for mapping and snp calling [\#57](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/57) ([andrewjpage](https://github.com/andrewjpage))
- tests pass when config variable output order changes [\#56](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/56) ([andrewjpage](https://github.com/andrewjpage))
- time to localtime to give sensible, human readable values [\#55](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/55) ([aslett1](https://github.com/aslett1))
- Constraint on tophat library type [\#54](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/54) ([js21](https://github.com/js21))
- Fixing bug RT:378950 [\#53](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/53) ([js21](https://github.com/js21))
- Tophat library\_type options [\#52](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/52) ([js21](https://github.com/js21))
- set default number of threads for eukaryotes [\#51](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/51) ([andrewjpage](https://github.com/andrewjpage))
- adapter removal for euk assembly [\#50](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/50) ([andrewjpage](https://github.com/andrewjpage))
- increase number of snp calling jobs run in each iteration [\#49](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/49) ([andrewjpage](https://github.com/andrewjpage))
- add user params for tophat [\#47](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/47) ([CraigPorter](https://github.com/CraigPorter))
- Single cell assembly [\#45](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/45) ([martinghunt](https://github.com/martinghunt))
- Base root and log [\#44](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/44) ([CraigPorter](https://github.com/CraigPorter))
- use pathnfs05 [\#43](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/43) ([andrewjpage](https://github.com/andrewjpage))
- Set bwa for bacteria rnaseq [\#42](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/42) ([CraigPorter](https://github.com/CraigPorter))
- assembly for eukaryotes [\#41](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/41) ([andrewjpage](https://github.com/andrewjpage))
- Add pairtype option for smalt [\#40](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/40) ([CraigPorter](https://github.com/CraigPorter))
- added config length filtering default and update assembly version number [\#39](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/39) ([martinghunt](https://github.com/martinghunt))
- specify pathogens gcdepth R script for QC [\#38](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/38) ([andrewjpage](https://github.com/andrewjpage))
- update default num threads for spades and velvet to 2 [\#37](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/37) ([andrewjpage](https://github.com/andrewjpage))
- GPL [\#36](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/36) ([andrewjpage](https://github.com/andrewjpage))
- add tag regex for file of lanes [\#35](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/35) ([CraigPorter](https://github.com/CraigPorter))
- different annotation config file per assembly [\#34](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/34) ([andrewjpage](https://github.com/andrewjpage))
- virus annotation [\#33](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/33) ([andrewjpage](https://github.com/andrewjpage))
- temp turn off assembly and annotation for all viruses [\#32](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/32) ([andrewjpage](https://github.com/andrewjpage))
- scripts to perform annotation and assembly [\#31](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/31) ([andrewjpage](https://github.com/andrewjpage))
- annotation, assembly and farm3 updates [\#30](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/30) ([andrewjpage](https://github.com/andrewjpage))
- Assembly and annotation limits work [\#29](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/29) ([CraigPorter](https://github.com/CraigPorter))
- updated usage text [\#28](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/28) ([CraigPorter](https://github.com/CraigPorter))
- SMALT version change to 0.7.4 for tests [\#27](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/27) ([aslett1](https://github.com/aslett1))
- SMALT version change to 0.7.4 [\#26](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/26) ([aslett1](https://github.com/aslett1))
- return instead of die for references in bin [\#25](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/25) ([andrewjpage](https://github.com/andrewjpage))
- use regex for lanes without specified sequence tag [\#24](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/24) ([CraigPorter](https://github.com/CraigPorter))
- tophat mark\_duplicates defaults to off [\#23](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/23) ([CraigPorter](https://github.com/CraigPorter))
- add sga error correction to assembly config [\#22](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/22) ([andrewjpage](https://github.com/andrewjpage))
- Add default memory for annotate assembly conf [\#21](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/21) ([andrewjpage](https://github.com/andrewjpage))
- versioning for annotation and assembly pipelines [\#20](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/20) ([andrewjpage](https://github.com/andrewjpage))
- specific version of bwa for qc pipeline [\#19](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/19) ([andrewjpage](https://github.com/andrewjpage))
- uppercase prokka [\#18](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/18) ([andrewjpage](https://github.com/andrewjpage))
- Add annotate assembly pipeline [\#17](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/17) ([andrewjpage](https://github.com/andrewjpage))
- exit instead of die when looking up references [\#16](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/16) ([andrewjpage](https://github.com/andrewjpage))
- dont backslash out lane names [\#15](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/15) ([andrewjpage](https://github.com/andrewjpage))
- update snp calling regex [\#14](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/14) ([andrewjpage](https://github.com/andrewjpage))
- Tests also now use SMALT v0.7.2 [\#13](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/13) ([aslett1](https://github.com/aslett1))
- SMALT version change to 0.7.2 [\#12](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/12) ([aslett1](https://github.com/aslett1))
- update warehouse host [\#11](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/11) ([andrewjpage](https://github.com/andrewjpage))
- mapping config limits escaped out like all other pipelines [\#10](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/10) ([andrewjpage](https://github.com/andrewjpage))
- make it harder to overwrite config files [\#9](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/9) ([andrewjpage](https://github.com/andrewjpage))
- connection details [\#8](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/8) ([andrewjpage](https://github.com/andrewjpage))
- Allow smalt parameters to be changed [\#7](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/7) ([andrewjpage](https://github.com/andrewjpage))
- Add bowtie2 and stampy for all tasks [\#6](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/6) ([andrewjpage](https://github.com/andrewjpage))
- Update stampy default to pathogens version [\#5](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/5) ([andrewjpage](https://github.com/andrewjpage))
- cast as hash [\#4](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/4) ([andrewjpage](https://github.com/andrewjpage))
- dont allow duplicate study registrations [\#3](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/3) ([andrewjpage](https://github.com/andrewjpage))
- hard code paths to libraries [\#2](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/2) ([andrewjpage](https://github.com/andrewjpage))
- Initial working script generation [\#1](https://github.com/sanger-pathogens/Bio-VertRes-Config/pull/1) ([andrewjpage](https://github.com/andrewjpage))


